### PR TITLE
Events detailView fixes

### PR DESCRIPTION
**Pull request by landongerrits**
screenshot 2018-10-14 00 55 51
 
tapping on location will grab correct address instead of search for the title of the location string label 
-  *Important note* - ON FIREBASE the full address must be in the "location" field and the title of the location must be in "locationTitle" field

location icon added next to location title

end date added to view (I still need to remove end date if the event is only one day long)

### CODE REVIEW

**Comment by landongerrits** [comment]
screenshot 2018-10-14 03 17 52


added start time

fixed issue #221 spacing btwn text on location

File: CruCentralCoast/Events/EventDetailsVC.swift [hunk 1]
```diff
@@ -39,9 +39,15 @@ class EventDetailsVC: UIViewController {
...
-            startEndDateArray.removeLast()
+        // if endDate time is less than 12hrs away, remove it
+        if let start = self.event?.startDate {
+            if let end = self.event?.endDate {
+                if start.timeIntervalSince(end) < 43200 {
+                    startEndDateArray.removeLast()
+                }
+            }
         }
+
```
Comment (tydahlwave) [inline CruCentralCoast/Events/EventDetailsVC.swift:48]: Why so much whitespace?

File: CruCentralCoast/Events/EventDetailsVC.swift [hunk 2]
```diff
@@ -39,9 +39,15 @@ class EventDetailsVC: UIViewController {
         guard let endDate = self.event?.endDate.toString(dateStyle: .medium, timeStyle: .none) else { return }
         var startEndDateArray = [startdate, endDate]
 
-        if (self.event?.endDate)!.timeIntervalSince((self.event?.startDate)!) < 43200{
-            startEndDateArray.removeLast()
+        // if endDate time is less than 12hrs away, remove it
+        if let start = self.event?.startDate {
```
Comment (tydahlwave) [inline CruCentralCoast/Events/EventDetailsVC.swift]: You can chain these two conditional statements `if let start = self.event?.startDate, let end = self.event?.endDate {`

File: CruCentralCoast/Events/EventDetailsVC.swift (hunk 1 above)
Comment (landongerrits) [reply CruCentralCoast/Events/EventDetailsVC.swift:48]: whoops, ya im not sure

File: CruCentralCoast/Events/EventDetailsVC.swift (hunk 2 above)
Comment (landongerrits) [reply CruCentralCoast/Events/EventDetailsVC.swift]: nice, thank you

**Comment by tydahlwave** [comment]
I think you should leave the date label as uppercase until we have a group discussion about it.

**Review by tydahlwave — APPROVED** [review]

**Pull request merged**